crates/oxc_semantic/tests/util/mod.rs

```diff
@@ -108,12 +110,22 @@ impl SemanticTester {
         SymbolTester::new_at_root(self, self.build(), name)
     }
 
+    /// Tests that a class with the given name exists
+    ///
+    /// ## Fails
+    /// If no class with the given name exists.
+    #[allow(dead_code)]
```

Comment:
Do we need dead_code in the tester?

Reply:
This line can be deleted, but dead_code is needed in ClassTester. I don't know why
<img width="565" alt="image" src="https://github.com/oxc-project/oxc/assets/29533304/eebeab75-7a67-49e7-b968-6a570d4a5ba7">

Reply:
Oh, they are only used in #[test] ... try add #[test] to these methods?